src/lib.rs

```diff
@@ -46,8 +50,11 @@ pub fn edit_distance(a: &str, b: &str) -> i32 {
                 // match or substitution
                 matrix[i][j] + if ca == cb { 0 } else { 1 }];
             matrix[i+1][j+1] = *alternatives.iter().min().unwrap();
+            j = j + 1;
         }
+        j = 0;
+        i = i + 1;
```

Comment:
@skade – I'm kinda unhappy about manually driving the loop counters here... You know any better way? `char_indices` would be just the thing, but the index is byte offset.